### ISSUE TITLE
Add fullscreen button to ModBar

### ISSUE THREAD

**Issue by p1-rde** [plotly/plotly.js#6287]
Sometimes when we have too many graphs on the page we seek to see each graph on fullscreen.
I have developed it already you can see it here https://gist.github.com/p1-rde/2120938f1c7f44c26755cfd83f8562e8

**Comment by Souravpakhira** [comment]
hey i was actually for it for can i get fullscreen mode. So can you help me in what files do i need to make changes to achieve fullscreen mode. thank you

**Comment by p1-rde** [comment]
> 

check this I have shared it on Gist  and don't forget to give a star https://gist.github.com/p1-rde/2120938f1c7f44c26755cfd83f8562e8

**Comment by Tofi666** [comment]
Also missing full screen option for plots so +1

**Comment by allaireetienne** [comment]
@Tofi666 how did you achieve to modify your dash app code to include the fullscreen option with the gist? Custom CSS file?

**Comment by allaireetienne** [comment]
> hey i was actually for it for can i get fullscreen mode. So can you help me in what files do i need to make changes to achieve fullscreen mode. thank you

Which files did you modify to achieve the fullscreen mode? I try this with python and dash.

**Comment by LeoDiep** [comment]
waiting this for so long. can you help me how to implement your code @p1-rde

**Comment by LeoDiep** [comment]
@gvwilson thank you a lot for your work you will put on this feature. I really appreciate it

**Comment by auxym** [comment]
Looking forward to this :)

**Comment by LeoDiep** [comment]
couldn't wait for it :)

**Comment by Ferreira-B** [comment]
I also agree that this is an important feature to be included in Dash applications, especially when one has a dashboard with several graphs and aims to expand each graph to full-screen temporarily.

**Comment by LeoDiep** [comment]
Hi, is this feature live yet. I haven't check the lastest version yet. Thanks

**Comment by AlbertoFabbri93** [comment]
This would also be very useful when using Quarto as the plots are limited by the width of the notebook.
